binding-service-impl/src/test/java/io/mosip/esignet/KeyBindingServiceTest.java

```diff
@@ -119,6 +135,24 @@ public void sendBindingOtp_withInvalidRequest_thenFail() throws SendOtpException
 		keyBindingService.sendBindingOtp(otpRequest, headers);
 	}
 
+
+	@Test
+	public void sendBindingOtpV2_withInvalidCaptcha_thenFail() throws SendOtpException {
+
+		BindingOtpRequestV2 otpRequest = new BindingOtpRequestV2();
+		otpRequest.setIndividualId("8267411571");
+		otpRequest.setOtpChannels(Arrays.asList("OTP"));
+		otpRequest.setCaptchaToken("qwerty");
+
+		Map<String, String> headers = new HashMap<>();
+
+		try {
+			keyBindingService.sendBindingOtpV2(otpRequest, headers);
+		} catch (EsignetException e) {
+			Assert.assertTrue(e.getErrorCode().equals(SEND_OTP_FAILED));
```

Comment:
If the request is sent with an invalid captcha, the error code to be asserted is "INVALID_CAPTCHA." How is the "SEND_OTP_FAILED" error code returned?